src/infill/LightningTreeNode.cpp

```diff
@@ -37,6 +37,23 @@ const Point& LightningTreeNode::getLocation() const
     return p;
 }
 
+Point LightningTreeNode::getFurthestLocation() const
+{
+    Point result{ p };
+    coord_t furthest_dist2 = 0;
+    for (const auto& child : children)
+    {
+        const Point cp{ child->getFurthestLocation() };
+        const coord_t dist2 = vSize2(cp - result);
```

Comment:
This will measure the distance between the current child's farthest leaf and the previously farthest leaf, rather than between the current child and the node itself. I think you meant this:
```suggestion
        const coord_t dist2 = vSize2(cp - p);
```